### PR TITLE
label.by_a_name rename and label.projects vs. label.donate fix

### PR DESCRIPTION
**Pull request by rabiul042**
Fix https://github.com/Plant-for-the-Planet-org/treecounter-app/issues/1666
Fix https://github.com/Plant-for-the-Planet-org/treecounter-app/issues/1644

### CODE REVIEW

File: app/components/PlantProjects/PlantProjectFull.js [hunk 1]
```diff
@@ -143,7 +143,7 @@ class PlantProjectFull extends React.Component {
             <div className="row">
               <div className="teaser__tpoHeading">
                 <a onClick={() => this.updateRoute(teaserProps.tpoSlug)}>
-                  {i18n.t('label.by_a_name') + ' ' + teaserProps.tpoName}
+                  {i18n.t('label.comp_by_name') + ' ' + teaserProps.tpoName}
```
Comment (norbertschuler) [inline app/components/PlantProjects/PlantProjectFull.js]: Using the resource `comp_by_name` without setting its both parameter append a space before and after the resource, e.g. ` by `.

Comment (mahmed0715) [reply app/components/PlantProjects/PlantProjectFull.js]: true @norbertschuler 
@rabiul042 please create a new resource that is only for the text 'by'

Comment (norbertschuler) [reply app/components/PlantProjects/PlantProjectFull.js]: no, I already changed it to use at least one placeholder from `comp_by_name` which also works well as just has a space in front which does not matter in HTML.

**Comment by norbertschuler** [comment]
> There is a use case where @sagararyal wanted to show Screen heading as 'Projects" for the donate trees, thus I needed to change that.
> @sagararyal please confirm.

O.k. sorry, then we have to change the used resource from `label.donate` to `label.projects` in the code. Do not change again the content of the resources.

**Comment by norbertschuler** [comment]
@mahmed0715 please merge if you think everything is done.

**Comment by mahmed0715** [comment]
Thanks

**Pull request merged**